package-dev/Runtime/SentryInitialization.cs

```diff
@@ -1,6 +1,10 @@
 using UnityEngine;
 using UnityEngine.Scripting;
 
+#if UNITY_IOS && !UNITY_EDITOR
```

Comment:
No idea if this works:
```suggestion
#define ENABLE_ISO_BRIDGE = UNITY_IOS && !UNITY_EDITOR
#if ENABLE_ISO_BRIDGE
```